packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -648,9 +648,17 @@ export async function* OpenAIChatModel(
               argsJson += toolCall.function.arguments;
             }
 
-            yield (
-              <FunctionCall id={id} partial name={name} args={JSON.parse(patchedUntruncateJson(argsJson || '{}'))} />
-            );
+            try {
+              yield (
+                <FunctionCall id={id} partial name={name} args={JSON.parse(patchedUntruncateJson(argsJson || '{}'))} />
+              );
```

Comment:
Try around the yield has other potential side effects, probably better to just wrap the untruncate call